### ISSUE TITLE
devfile v1 -> v2 convertion enchancements 

### ISSUE THREAD

**Issue by ibuziuk** [eclipse/che#21103]
### Is your enhancement related to a problem? Please describe

UX / Docs

- [x]  Action dropdown should not exist since there is only one action / if the only action is delete, lets make it a button /

![image](https://user-images.githubusercontent.com/1461122/153001516-8947745c-5fec-4d47-ac54-6976b995f95f.png)

- [x]  Once the conversion is done successfully "Convert" button shd gone away, There shd not be a button and then a error message (The conversion button should not be active the second time but we should show the delete)
- [ ]  Documentation must be clear as to what is the difference between devfile1 Vs devfle2 and what changes will the user see 
       - N.B. There is a separate issue for documentation https://github.com/eclipse/che/issues/21005 **(TODO discuss on cabal)**
       - Specify clearly what possible error messages can happen and what they need to do on CRW (?) docs **(TODO discuss on cabal)**
       - [ ] Add FAQ / Known issues with volumes to devfile.io / update the link from UD to FAQ (action prepare content)
- [x]  Can we find a way to warn if they will succeed in conversion? and warn them upfront. **(TODO discuss on cabal)** (cover as part of docs FAQ / Known issues)
- [x]  In the warning you should specify that it’s possible to convert it automatically with the button above. (Convert, than look to FAQ if is failing) https://github.com/eclipse-che/che-dashboard/pull/457

Features:

- [x] https://github.com/eclipse/che/issues/21153

Bugs
- [x] https://github.com/eclipse/che/issues/21197
- [x] https://github.com/eclipse/che/issues/21198
- [x] https://github.com/eclipse/che/issues/21120
- [x] https://github.com/eclipse/che/issues/21121

### Describe the solution you'd like

N/A

### Additional context

The initial convention mechanism has been implemented as part of https://github.com/eclipse/che/issues/20845

**Comment by ibuziuk** [comment]
@l0rd updated deprecated note PR - https://github.com/eclipse-che/che-dashboard/pull/457

**Comment by ibuziuk** [comment]
Closing the issue since we have 2 separate tickets for docs and the conversion enhancements:
- https://github.com/eclipse/che/issues/21005
- https://github.com/eclipse/che/issues/21120
